parliamentarian/__init__.py

```diff
@@ -38,7 +45,7 @@ def override_config(override_config_path):
 
 def enhance_finding(finding):
     if finding.issue not in config:
-        raise Exception("Uknown finding issue: {}".format(finding.issue))
+        raise Exception("Unknown finding issue: {}".format(finding.issue))
```

Comment:
nit: you can opportunistically update these to f'strings since it is being touched